extra/rest_client.rb

```diff
@@ -2,8 +2,32 @@ module Oxidized
   class RestClient
     require 'net/http'
     require 'json'
-    HOST = 'localhost'
-    PORT = 8888
+    require 'uri'
+    require 'asetus'
+
+    class Config
+      Root      = File.join ENV['HOME'], '.config', 'oxidized'
+    end
+
+    CFGS = Asetus.new :name=>'oxidized', :load=>false, :key_to_s=>true
+    CFGS.default.rest = '127.0.0.1:8888'
+
+    begin
+      CFGS.load
+    rescue => error
+      raise InvalidConfig, "Error loading config: #{error.message}"
+    ensure
```

Comment:
following [your comment](https://github.com/ytti/oxidized/pull/467#issuecomment-226261488), you don't need this `ensure`.

``` ruby
    begin
      CFGS.load
    rescue => error
      raise InvalidConfig, "Error loading config: #{error.message}"
    end

    restcfg = CFGS.cfg.rest
```